File: Codewars/7_kyu_smallest_product/Akiekano/Solutions.py
#  SPDX-License-Identifier: GPL-2.0-only
# 
#   Copyright (C) 2021  Akiekano <hello.akiekano@gmail.com> 
def smallest_product(a):
    array = []
    for x in a:
        dec = 1
        for y in x:
            dec = dec*y
        array.append(dec)
    return min(array)

File: Codewars/7_kyu_factorial/Akiekano/Solutions.go
// SPDX-License-Identifier: GPL-2.0-only
/*
 * Copyright (C) 2021  Akiekano <hello.akiekano@gmail.com>
 */
package kata

func Factorial(n int) int {
  if n<=1 {
    return 1
  }
  return Factorial(n-1) * n
}